notebooks/EVA-release-RSIDquery.ipynb

```diff
@@ -0,0 +1,179 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Our general query provides an abundance of information including date the ID was created and if the variant has been remapped to a newer assembly. For this task we want to find the associated clustered SS IDs, species/assembly, & variant location"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can parse the JSON output to find data of interest and assign these to variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ss_list=[]\n",
+    "for record in output:\n",
+    "    ssroll = record['accession']\n",
+    "    ss_list.append(ssroll)\n",
```

Comment:
Not important but what's the variable name `ssroll` mean here?  If you want to be fancy you can also just do `ss_list.append(record['accession'])`